cmd/workflow.go

```
@@ -1,40 +1,198 @@
 	Run: func(cmd *cobra.Command, args []string) {
+		// If no arguments are provided, start the workflow UI
+		if len(args) == 0 {
+			err := e.ExecuteWorkflowCmd(cmd, args)
+			if err != nil {
+				u.LogErrorAndExit(schema.AtmosConfiguration{}, err)
+			}
+			return
+		}
+
+		if args[0] == "help" {
+			cmd.Help()
```

golangci-lint (1.62.2)

Error return value of `cmd.Help` is not checked

(errcheck)